app/javascript/components/DownloadButton.js

```diff
@@ -115,6 +145,59 @@ function DownloadCommandContainer(props) {
   )
 }
 
+/**
+ * Format number in bytes, with human-friendly units
+ *
+ * Derived from https://gist.github.com/lanqy/5193417#gistcomment-2663632
+ */
+function bytesToSize(bytes) {
+  const sizes = ['bytes', 'KB', 'MB', 'GB', 'TB']
```

Comment:
I don't know what sizes of downloads are customary, but it might be helpful to NOT show  things in TB, and instead show as, e.g. `1,300GB`,so that it's more noticeable how large it is